app/index.js

```diff
@@ -149,37 +155,33 @@ var NemoGenerator = yeoman.generators.Base.extend({
       this.sauceSetup = props.sauceSetup;
       this.sauceUser = props.sauceUser;
       this.sauceKey = props.sauceKey;
+      this.testFramework = props.testFramework;
       this.deployedUrl = (!!props.deployedUrl) ? props.deployedUrl : undefined;
       done();
     }.bind(this));
   },
   editGruntfile: function() {
-    var that = this;
-    var done = this.async();
-
-    //let's update Gruntfile.js if possible
-    fs.readFile('Gruntfile.js', 'utf8', function(err, data) {
-
-      if (err) {
-        return console.log(err);
+      var that = this,
+          done = this.async(),
+          exists = fs.existsSync('Gruntfile.js');
+      if (!exists) {
+        var gruntContent = 'module.exports = function (grunt) { require(\'grunt-config-dir\')(grunt, {configDir: require(\'path\').resolve(\'tasks\')});};';
+        fs.writeFileSync('Gruntfile.js',gruntContent);
       }
-      if (data.match(/loopmocha/) !== null) {
-        //already here?
-        done();
-        return;
+    //let's update Gruntfile.js if possible
+    var api = require('gruntfile-api'),
```

Comment:
Please move require to top of file and give more descriptive variable name like gruntfileAPI. Will help to clarify what's going on for folks coming to this fresh.